vmdb/db/migrate/20150405141637_remove_port_config_from_container_service.rb

```diff
@@ -0,0 +1,54 @@
+class RemovePortConfigFromContainerService < ActiveRecord::Migration
+  def up
+    create_table :container_service_port_configs do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.integer    :port
+      t.string     :target_port
+      t.string     :protocol
+      t.belongs_to :container_service, :type => :bigint
+    end
+
+    copy_over_configs
+    remove_columns
+  end
+
+  def copy_over_configs
+    ContainerService.find_each do |service|
+      ContainerServicePortConfig.create(
```

Comment:
- Avoid using models directly in migrations.  You will have to use stubs for data access.
- Migration specs are required for data migrations.
- When you have a section with a data migration wrap it in a `say_with_time` block.  You can see that in other data migrations, and it's used to show progress and time to complete in the migration output.